packages/@aws-cdk/aws-glue-alpha/lib/job-executable.ts

```diff
@@ -87,6 +87,16 @@ export enum PythonVersion {
   THREE_NINE = '3.9',
 }
 
+/**
+ * Runtime
+ */
+export enum Runtime {
+  /**
+   * Ray 2.4
+   */
+  RAY_TWO_FOUR = 'Ray2.4',
```

Comment:
This would be a repetitive operation, wondering if we want to move away from enums

Reply:
should be an enum-like class:

```ts
export class Runtime {
  public static RAY_TWO_FOUR = new Runtime('Ray2.4');
  public constructor(public readonly runtime: string) {}
}
```

Reply:
Thanks for sharing the example. Reflected with minor change.